### DISCUSSION TITLE
Poll: UTM enhancement to IMM mode

### DISCUSSION THREAD

**Discussion by paregupt** [paregupt/ucs_traffic_monitor#105] [Polls]
UTM only works in UCSM mode, not in IMM. Also, Intersight now supports performance monitoring. Please share your thoughts to decide UTM's future enhancements.
Please feel free to add more details in addition to answering the poll.

**Comment by eminchen** [comment]
FYI

As of this week, Intersight SaaS Metrics explorer, in addition to IMM mode, now supports UCSM domains as well.

https://intersight.com/help/saas/features/monitoring/monitoring_metrics_explorer

Regards,
Eugene

On Sep 21, 2024, at 10:27 AM, Paresh Gupta ***@***.***> wrote:

﻿

UTM only works in UCSM mode, not in IMM. Also, Intersight now supports performance monitoring. Please share your thoughts to decide UTM's future enhancements.
